presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -834,6 +834,16 @@ Log the stats equivalent plan and canonicalized plans used in history based opti
 Enable analysis and propagation of logical properties like distinct keys or cardinality among the nodes of
 a query plan. The optimizer may then use these properties to perform various optimizations.
 
+``optimizer.confidence-based-broadcasting``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+Enable broadcasting based on the confidence of the statistics that are being used, specifically choosing to
```

Comment:
```suggestion
Enable broadcasting based on the confidence of the statistics that are being used, by
```

Reply:
Thanks for the comments! Making the changes now

Reply:
Sorry just made a small change, based on something I changed recently. should be good to go now!